### ISSUE TITLE
where's the problem this time???

### ISSUE THREAD

**Issue by Urbexed** [Nathoune987/UptoDL#2]
![Capture d’écran (5)](https://github.com/spel987/UptoDL/assets/116701347/02a3a506-106d-4990-a2fa-f9c364d1dc41)

**Comment by Natthanael** [comment]
Hello i got the same problem . 

Thanks in advance

**Comment by Urbexed** [comment]
> Hello i got the same problem .
> 
> Thanks in advance

At least it’s not coming from my PC this time 😴

**Comment by spel987** [comment]
Hello, 

For the selenium problem, simply uninstall selenium :

`pip uninstall selenium -y`
`pip uninstall seleniumwire -y`

And reinstall with an earlier version: 

`pip install selenium==4.9.1`.

This will work, but you won't be able to download your content. Uptbox has added a captcha system (with Cloudflare). Tor proxies make captchas recurrent. 

In addition to these captchas, Uptbox has changed the way it loads videos. This has made it difficult, if not impossible, to download these videos. There's a kind of token and an anti-selenium. 

I don't know if I can fix these two problems, especially as I don't have much time at the moment. I'll specify on the repository page that it's deprecated.

**Comment by Natthanael** [comment]
![image](https://github.com/spel987/UptoDL/assets/124980485/95512851-4cf5-497a-af1a-9894fcefbe85)

So all of this it's normal  ?

**Comment by spel987** [comment]
Yes, the script cannot retrieve the MPD file because it has changed.
